change/@fluentui-react-timepicker-compat-5fe049e3-8f00-497c-b67f-a8a48b779afa.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Preview release",
```

```suggestion
  "comment": "feat: release compat package",
```

https://github.com/microsoft/fluentui/blob/2a42b52a62aa5bc6e2c0fb81e43d4edf0b0fc4e9/tools/workspace-plugin/src/generators/prepare-initial-release/index.spec.ts#L166-L167